.github/workflows/ci_cd.yml

```diff
@@ -198,26 +198,25 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      # ℹ️Step only applies for frontend and catalog images.
-      # Frontend downloads translations and catalog runs `just` recipes.
+      # Frontend downloads translations.
```

Comment:
Would be nice to add that this step sets the python/node/pnpm versions

Reply:
This step does not do that.